README.md

```diff
@@ -20,6 +20,50 @@ money_column expects a DECIMAL(21,3) database field.
 
     gem 'shopify-money', require: 'money'
 
+## Road to v1.0
```

Comment:
Maybe an `upgrading.md` instead and only use the readme to show the 'good' behaviour? It can get a little overwhelming otherwise.